### ISSUE TITLE
Error when go to definition of PyQt5's module

### ISSUE THREAD

**Issue by ppaanngggg** [Microsoft/python-language-server#440]
I have code like following
```python
from PyQt5.QtChart import QChart, QChartView, QLineSeries, QValueAxis
from PyQt5.QtWidgets import QApplication, QTabWidget, QVBoxLayout, QWidget
```

When I go to the definition of `PyQt5.QtChart` or `PyQt5.QtWidgets`, then VSCode popup
the OUTPUT panel and print like following

> [Error - 7:18:34 PM] Request textDocument/definition failed.
  Message: Value cannot be null.
Parameter name: source
  Code: -32000 
   at System.Linq.Enumerable.Select[TSource,TResult](IEnumerable`1 source, Func`2 selector)
   at Microsoft.Python.LanguageServer.Implementation.Server.GetModuleReferences(IPythonProjectEntry entry, PythonAst tree, BufferVersion version, ReferencesParams params)
   at Microsoft.Python.LanguageServer.Implementation.Server.FindReferences(ReferencesParams params, CancellationToken cancellationToken)
   at Microsoft.Python.LanguageServer.Implementation.Server.GotoDefinition(TextDocumentPositionParams params, CancellationToken cancellationToken)
   at Microsoft.Python.LanguageServer.Implementation.LanguageServer.GotoDefinition(JToken token, CancellationToken cancellationToken)

**Comment by MikhailArkhipov** [comment]
Needs null check

**Comment by ppaanngggg** [comment]
Thanks :D

**Comment by jakebailey** [comment]
Fixed in #498. Should come to the daily/beta/insiders channels later tonight.
